packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -312,29 +312,6 @@ export const installGovAndPSMContracts = async ({
   );
 };
 
-/**
- * @deprecated the PSM charter has been merged with the econ charter
- * @param {EconomyBootstrapPowers} powers
- */
-export const startPSMCharter = async ({
```

Comment:
Isn't that still used by `packages/vats/src/core/boot-psm.js`? Ah... not as of fa9c030d27839d